internal/buildscripts/packaging/installer/install.sh

```diff
@@ -871,12 +871,11 @@ Auto Instrumentation:
                                         applications running as systemd services.
                                         Cannot be combined with the '--with-instrumentation' option.
                                         (default: --without-systemd-instrumentation)
-  --with[out]-java-instrumentation      Whether to enable Auto Instrumentation for Java.
-                                        (default: --with-java-instrumentation if --with-instrumentation or
+  --with[out]-instrumentation-sdk "<s>" Whether to enable Auto Instrumentation for a specific language. This option
+                                        takes a comma separated set of values representing supported
+                                        auto-instrumentation SDKs. Currently supported: java,node.
+                                        (default: --with-instrumentation-sdk "java,node" if --with-instrumentation or
                                         --with-systemd-instrumentation is also specified)
```

Comment:
Q not related to this change: if systemd instrumentation the user has to put some info to enable each instrumentation, right? I mean there is data in systemd that triggers each instrumentation, no? Here it sounds like that just adding `--with-systemd-instrumentation` you will get java and node instrumentations, without any other action. Is this correct?

Reply:
the only difference between the 2 modes is how we inject the env vars. With the /etc/preload.so approach, we have a bit of C code that detects the process and adds the env vars dynamically. With the systemd approach, we add env vars as a config value for systemd services. The downside of /etc/preload.so is the security aspect of intercepting program invocation. The downside of systemd is that it only applies to systemd services. I expect customers can find something that works for them based on these compromises.

Reply:
ah, thanks - I misunderstood how instrumentation via systemd was configured.